packages/api/cms-api/src/user-permissions/user-permissions.const.ts

```diff
@@ -0,0 +1 @@
+export const USER_PERMISSIONS_CONFIG_SERVICE = "userpermissions-config-service";
```

Comment:
This file should be called user-permissions.constants.ts to be in line with the other modules